crates/graphql/src/schema/objects/chart.rs

```diff
@@ -1,12 +1,13 @@
 use indexer_core::db::queries::charts;
-use objects::auction_house::AuctionHouse;
+use objects::{auction_house::AuctionHouse, creator::Creator};
 use scalars::{PublicKey, U64};
 
 use super::prelude::*;
 
 #[derive(Debug, Clone)]
 pub struct PriceChart {
     pub auction_houses: Vec<PublicKey<AuctionHouse>>,
+    pub creators: Option<Vec<PublicKey<Creator>>>,
```

Comment:
Any reason this isn't just a `Vec<Creator>`?

Reply:
@ray-kast Creators field can be null in case we want to fetch charts data for the whole marketplace. And in case of a particular creater's chart data creator field needs to be populated. So, kept it optional.

Reply:
Yep. If no creators provided returned stats for the whole auction_house or houses.

Reply:
Ah, it's a query parameter.  Got it.